.buildkite/scripts/triggerJenkinsJob/jenkins.go

```diff
@@ -0,0 +1,96 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/bndr/gojenkins"
+	"github.com/pkg/errors"
```

Comment:
This package is not maintained, use `errors` in the standard library.

Reply:
hmm if I changed to "errors", I have these errors when running `make build`

```
.buildkite/scripts/triggerJenkinsJob/jenkins.go:25:22: undefined: errors.Wrap
.buildkite/scripts/triggerJenkinsJob/jenkins.go:59:22: undefined: errors.Wrapf
.buildkite/scripts/triggerJenkinsJob/jenkins.go:64:22: undefined: errors.Wrapf
```

Reply:
hmm If I changed to `errors`, I have these errors when running the script:
```
 $ go run ./.buildkite/scripts/triggerJenkinsJob -h
# github.com/elastic/elastic-package/.buildkite/scripts/triggerJenkinsJob
.buildkite/scripts/triggerJenkinsJob/jenkins.go:25:22: undefined: errors.Wrap
.buildkite/scripts/triggerJenkinsJob/jenkins.go:59:22: undefined: errors.Wrapf
.buildkite/scripts/triggerJenkinsJob/jenkins.go:64:22: undefined: errors.Wrapf
.buildkite/scripts/triggerJenkinsJob/jenkins.go:92:22: undefined: errors.Wrapf
```

Reply:
Got it, the errors module from the standard library does not have those methods. Using `fmt.Errorf` instead of those.